Added note on support for PolicyBasedTrafficSelectors

ScottNapolitan opened the pull request:
Azure Stack does not support usePolicyBasedTrafficSelectors for Route Based VPNs.

azurecla commented:
Hi __@prashanthmadi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@prashanthmadi : Thanks for your contribution to the Azure documentation! The author, @christopheranderson, has been notified to review your proposed change.

christopheranderson commented:
@lindydonna - Could you review these since you're the App Service Editor expert?

lindydonna commented on articles/azure-functions/functions-reference.md:
```diff
@@ -78,21 +78,19 @@ The function editor built into the Azure portal lets you update the *function.js
-5. Turn it **On** if it is not already enabled, and click **Go**.
+4.  click **Go**.
 
 	After Visual Studio Online loads, you'll see the *host.json* file and function folders under *wwwroot*. 
```
@prashanthmadi Thanks for your contribution!  Do you mind also changing the reference on line 91?  If not, we'll accept the PR and fix ourselves :)

prashanthmadi replied:
@lindydonna , yes i have made that change.

lindydonna commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

christopheranderson commented:
#sign-off

ScottNapolitan committed:
Added note on support for PolicyBasedTrafficSelectors

Azure Stack does not support usePolicyBasedTrafficSelectors for Route Based VPNs.

ktoliver committed:
fix typo

ktoliver committed:
Merge branch 'master' into patch-3
